src/formElementPlugins/imageDisplay/pluginConfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "isCore": true,
+  "code": "imageDisplay",
+  "displayName": "Image Display",
+  "dynamicParameters": ["label", "description", "url", "size", "alignment"],
```

Comment:
```suggestion
  "dynamicParameters": ["label", "description", "url", "size", "alignment", "altText"],
```

Reply:
Again -- do you really think something like "altText" needs to be dynamic?

Reply:
Hmm maybe I didn't get this field correctly, I though it was listing fields that could be passed in to this plugin question

Reply:
No, this is just the list of fields that the Wrapper will assume could be evaluator expressions, so will run evaluator on them. Fields not listed here will be passed through to plugin as is.